### DISCUSSION TITLE
How do I remove the YAML Frontmatter section from my notes?

### DISCUSSION THREAD

**Discussion by scotrod** [Zettlr/Zettlr#4636] [Q&A] [answered]
Hey All, as I'm currently in the process of migrating my notes from Joplin over to Zettlr, I'm having an issue where I cannot locate where is the setting for disabling this particular section from my notes:

---
title: test
updated: 2022-11-14 15:54:17Z
created: 2022-10-13 13:25:47Z
latitude: 42.69770820
longitude: 23.32186750
altitude: 0.0000
---

This is being placed by Zettlr on the top of each note, and I'm wrapping my head trying to hide it.

Any ideas where should I look into?

**Comment by nathanlesage** [comment, marked as answer]
This looks like metadata generated by Joplin. I have used Joplin at some point in the past and realized that it apparently used the YAML frontmatter to store its metadata, and apparently does keep that on export. I have seen this on my exports from Joplin as well. It is definitely not data added by Zettlr -- you may be able to find settings in Joplin to disable that behavior, otherwise you could run a search and replace across all files to remove this info.

**Reply by scotrod** [reply]
You are right, thank you. I just used Joplin's export to export "markdown only" without the Front Matter:

![image](https://github.com/Zettlr/Zettlr/assets/45900128/b1781e37-59a2-48c5-a6a5-bd04eb50af25)
